oap-server/server-starter/src/main/resources/otel-rules/bookkeeper/bookkeeper-node.yaml

```diff
@@ -0,0 +1,90 @@
+#    "PT15M"     -- parses as "15 minutes" (where a minute is 60 seconds)
+#    "PT10H"     -- parses as "10 hours" (where an hour is 3600 seconds)
+#    "P2D"       -- parses as "2 days" (where a day is 24 hours or 86400 seconds)
+#    "P2DT3H4M"  -- parses as "2 days, 3 hours and 4 minutes"
+#    "P-6H3M"    -- parses as "-6 hours and +3 minutes"
+#    "-P6H3M"    -- parses as "-6 hours and -3 minutes"
+#    "-P-6H+3M"  -- parses as "+6 hours and -3 minutes"
+# </pre>
+filter: "{ tags -> tags.job_name == 'bookkeeper-monitoring' }" # The OpenTelemetry job name
+expSuffix: tag({tags -> tags.cluster = 'bookkeeper::' + tags.cluster}).instance(['cluster'], ['node'], Layer.BOOKKEEPER)
+metricPrefix: meter_bookkeeper_node
+
+# Metrics Rules
+metricsRules:
+  # thread executor metrics
+  - name: thread_executor_completed
+    exp: bookkeeper_server_thread_executor_completed.sum(['cluster', 'node'])
+  - name: thread_executor_tasks_completed
+    exp: bookkeeper_server_thread_executor_tasks_completed.sum(['cluster', 'node'])
+  - name: thread_executor_tasks_rejected
+    exp: bookkeeper_server_thread_executor_tasks_rejected.sum(['cluster', 'node'])
+  - name: thread_executor_tasks_failed
+    exp: bookkeeper_server_thread_executor_tasks_failed.sum(['cluster', 'node'])
+
+  - name: high_priority_threads
+    exp: bookkeeper_server_BookieHighPriorityThread_threads.sum(['cluster', 'node'])
+  - name: read_thread_pool_threads
+    exp: bookkeeper_server_BookieReadThreadPool_threads.sum(['cluster', 'node'])
+  - name: high_priority_thread_max_queue_size
+    exp: bookkeeper_server_BookieHighPriorityThread_max_queue_size.sum(['cluster', 'node'])
+  - name: read_thread_pool_max_queue_size
+    exp: bookkeeper_server_BookieReadThreadPool_max_queue_size.sum(['cluster', 'node'])
+
+  # JVM Metrics
+  - name: jvm_memory_used
+    exp: jvm_memory_bytes_used.sum(['cluster', 'node'])
+  - name: jvm_memory_committed
+    exp: jvm_memory_bytes_committed.sum(['cluster', 'node'])
+  - name: jvm_memory_init
+    exp: jvm_memory_bytes_init.sum(['cluster', 'node'])
+
+  - name: jvm_memory_pool_used
+    exp: jvm_memory_pool_bytes_used.sum(['cluster', 'node', 'pool'])
+  - name: jvm_memory_pool_committed
+    exp: jvm_memory_pool_bytes_committed.sum(['cluster', 'node', 'pool'])
+  - name: jvm_memory_pool_init
+    exp: jvm_memory_pool_bytes_init.sum(['cluster', 'node', 'pool'])
+
+  - name: jvm_buffer_pool_used_bytes
+    exp: jvm_buffer_pool_used_bytes.sum(['cluster', 'node', 'pool'])
+  - name: jvm_buffer_pool_capacity_bytes
+    exp: jvm_buffer_pool_capacity_bytes.sum(['cluster', 'node', 'pool'])
+  - name: jvm_buffer_pool_used_buffers
+    exp: jvm_buffer_pool_used_buffers.sum(['cluster', 'node', 'pool'])
+
+  - name: jvm_gc_collection_seconds_count
+    exp: jvm_gc_collection_seconds_count.sum(['cluster', 'node', 'gc']).rate('PT1M')
+  - name: jvm_gc_collection_seconds_sum
+    exp: jvm_gc_collection_seconds_sum.sum(['cluster', 'node', 'gc']).rate('PT1M')
```

Comment:
I think `rate` is not a good calculation for `gc seconds count/sum`. 
please check `rate` definition: https://skywalking.apache.org/docs/main/next/en/concepts-and-designs/mal/#rate